content/dstu2/authorization-specification.md

```diff
@@ -0,0 +1,1029 @@
+described previously in "Contextless Flow" to access resources.
+
+### SMART Application Launch Flow ###
+
+<pre class="terminal">                         
++--------------+                                         +--------------+
+|              |                                         |              |
+| Resource     |                                         | SMART        |
+| Owner        +--(1)------------------------------------> Application  |
+|              |                                         |              |
+|              |                                         |              |
+|              |            +---------------+            |              |
+|              |            |               |            |              |
+|              <--(5)-------+ Authorization <--(4)-------+              |
+|              |            | Server        |            |              |
+|              +--(6)------->               +--(7)------->              |
+|              |            |               |            |              |
++--------------+            |               <--(8)-------+              |
+                            |               |            |              |
+                            |               +--(9)------->              |
+                            |               |            |              |
+                            +---------------+            |              |
+                                                         |              |
+                            +---------------+            |              |
+                            |               |            |              |
+                            | FHIR Resource <--(2)-------+              |
+                            | Server        |            |              |
+                            |               +--(3)------->              |
+                            |               |            |              |
+                            |               <--(10)------+              |
+                            |               |            |              |
+                            |               +--(11)------>              |
+                            |               |            |              |
+                            +---------------+            +--------------+
+</pre>
+__Figure 3__: SMART Launch Flow Diagram
+
+In Figure 3, the following workflow is described:
+
+1.	The end user selects an option in a SMART application to retrieve 
+    context from the EHR application.
+2.	The SMART application performs [discovery](#discovery) by requesting the 
+    FHIR<sup>®</sup> server's conformance statement.  The mechanism for 
+	how the SMART application is provided the URL for the FHIR<sup>®</sup> 
+	server is not defined by this specification.
+3.  The FHIR<sup>®</sup> server returns the conformance statement, 
+    which provides the needed endpoints for steps 8 and 12.
```

Comment:
@MaxPhilips if you want to update this to steps 4 and 8, that'll correct the issue.  (If you also remove the May open-id connect page, that may resolve our outstanding issues.)